experimental/dds/ot/ot/src/test/delta.ts

```diff
@@ -10,7 +10,7 @@ import {
 	IChannelFactory,
 } from "@fluidframework/datastore-definitions";
 import Delta from "quill-delta";
-import { SharedOT } from "../../dist";
+import { SharedOT } from "../../dist/index.js";
```

Comment:
eh? a dist reference?

Reply:
Weird.  Here is what ../../dist/index.js looks like:
```
Object.defineProperty(exports, "__esModule", { value: true });
exports.SharedOT = void 0;
var ot_js_1 = require("./ot.js");
Object.defineProperty(exports, "SharedOT", { enumerable: true, get: function () { return ot_js_1.SharedOT; } });
//# sourceMappingURL=index.js.map
```

Reply:
That looks like ESM compatible CJS to me. The presence of package.json in dist allows it to be viable, but we shouldn't reference it without good reason and comments. Perhaps just some obsolete reason now.

Reply:
Yeah, I'm sure it was accidental...  VS Code will sometimes helpfully generate imports like this when the language service is confused/reparsing.  Fixed now.